.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 💡General Feature Request
+about: Have a new idea/feature for Image-Searcher? Please suggest!
+title: '[FEATURE] <description>'
+labels: []
+assignees: []
+---
+
+**Description**
+
+Please describe the Feature in detail.
+
+**Expected Behavior**
+
+What should happen?
+
+**Actual Behavior**
```

Comment:
As it is a new feature, I don't think it is necessary.

Reply:
cool will remove that 👍🏼